octodns_ultra/__init__.py

```diff
@@ -325,6 +328,19 @@ def populate(self, zone, target=False, lenient=False):
                 name = zone.hostname_from_fqdn(record['ownerName'])
                 if record['rrtype'] == 'SOA (6)':
                     continue
+                if (
+                    record['rrtype'] == 'A (1)'
+                    and 'profile' in record
+                    and record['profile']['@context']
+                    == 'http://schemas.ultradns.com/DirPool.jsonschema'
```

Comment:
It's the nittiest of nits, but the following would be slightly more succinct and avoid looking for `profile` multiple times.

```suggestion
                    and record.get('profile', {}).get('@context'])
                    == 'http://schemas.ultradns.com/DirPool.jsonschema'
```

Reply:
I'm not familiar with that syntax, but looks great.